### DISCUSSION TITLE
Bills overview

### DISCUSSION THREAD

**Discussion by poudenes** [firefly-iii/firefly-iii#8375] [Firefly III]
Hi All,

I have a question about the 2 values below the BILLS list. 

In my case (at moment of writing)
€ 1177,05
€ 1426,56

The total bills together paid and unpaid are € 1426,56
I don't know what 1177,05 tells me.

When I calculate all unpaid bills I get now € 994,86 (at moment of writing)
That's a difference of € 182,19 compared with the € 1177,05.

Questions:
1) what means the first amount ? (1177,05)?
2) if my idea of what it can be, why is there a difference?

Greetings,
Peter

**Comment by JC5** [comment]
I also don't know what these numbers mean. I can't see the transactions linked to the bill(s).

Amount 1: the total amount of expenses in the current period linked to a (any) bill.
Amount 2: the sum of all the average (amount min + amount max / 2) bill amounts yet unpaid.

**Reply by poudenes** [reply]
Amount 1: looks like more all unpaid bills
Amount 2: all bills together total

In my first post the 
Amount 1: 1177,05
Amount 2: 1426,56 (calculated all bills paid/unpaid and outcome was same)

Calculated unpaid bills for this period was 994,86 and not 1177,05

if you want I can send a screenshot (private) so you see what I mean.

**Reply by JC5** [reply]
Oh wait yeah no you're right. 

> Calculated unpaid bills for this period was 994,86 and not 1177,05. if you want I can send a screenshot (private) so you see what I mean.

A screenshot wouldn't really help me, I can look in the code to find what the math is exactly if you're interested.

**Reply by poudenes** [reply]
That will help. Because the manual calculation (994,86) is different then Firefly calculation (1177,05) what a difference make of  € 182,19. What else is calculated with Firefly and not by myself manual.

p.s. I also checked if there where any MIN/MAX amounts but total that was only € 2,-. Rest of the bills have a fixed amount

**Reply by JC5** [reply]
Is there a bill that could fall twice in the period?

**Reply by poudenes** [reply]
There is nothing that is 182,19. And no bill are twice in the list
